TestViewer/MainWindow.xaml

```diff
@@ -15,7 +15,7 @@
         <StackPanel Grid.Column="0">
             <!-- Menu pane -->
             <Label>Loader name</Label>
-            <TextBox x:Name="LoaderName">DummyLoader.Image3dFileLoader</TextBox>
+            <TextBox x:Name="LoaderName">GEHC_CARD_US.Image3dFileLoader</TextBox>
```

Comment:
Can you please keep "DummyLoader.Image3dFileLoader" as default value, so that the application works out-of-the-box also for users without access to the GE CVUS loader.

You can convert the TextBox into a pull-down menu or similar, where "GEHC_CARD_US.Image3dFileLoader" is one of the options, if you want to ease access to the GE CVUS loader. Please do that in a separate PR.